Makefile

```diff
@@ -132,6 +132,24 @@ docker-buildx: test ## Build and push docker image for the manager for cross-pla
 	- docker buildx rm project-v3-builder
 	rm Dockerfile.cross
 
+###########
+# Release #
+###########
+
+##@ release:
+
+export DISABLE_RELEASE_PIPELINE ?= true
+substitute:
+	envsubst < .goreleaser.template.yml > .goreleaser.yml
+
+release: GORELEASER_ARGS ?= --snapshot --rm-dist
+release: goreleaser substitute ## Run goreleaser
+	$(GORELEASER) $(GORELEASER_ARGS)
+
+quickstart: VERSION ?= $(shell git describe --abbrev=0 --tags)
+quickstart: generate ## Generate the installation release manifests
+	kubectl kustomize config/default | sed "s/:latest/:$(VERSION)/g" > operator-controller.yaml
```

Comment:
Kustomize can do image ref substitution directly. We may want to consider whether that is a viable alternative. If we aren't very careful, this approach could silently replace _other_ image refs in our manifests if they happen to use `:latest`.

Really _anything_ referencing `latest` in our manifests would be a problem, so maybe in practice this would be fine. But I'd rather not compound that problem if it does somehow happen.

Reply:
Yeah I can definitely see that being a problem. Let me fix this up to make sure we're just using the substitutions provided by kustomize.

Reply:
OK, with the latest push I have this being done through kustomize using the `VERSION` env set during release. Let me know what you think :)